File: services/monolith/src/test/java/se/tobiaslafleur/examplecg/service/monolith/SanityTest.java
package se.tobiaslafleur.examplecg.service.monolith;

import static org.assertj.core.api.Assertions.assertThat;

import org.junit.jupiter.api.Test;
import org.springframework.boot.test.context.SpringBootTest;

@SpringBootTest
class SanityTest {

    @Test
    void sanityCheck() {
        var expected = true;
        assertThat(expected).isTrue();
    }
}

File: services/monolith/src/test/kotlin/se/tobiaslafleur/examplecg/service/monolith/SanityTest.kt
package se.tobiaslafleur.examplecg.service.monolith

import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

class SanityTest {
    @Test
    fun sanityCheck() {
        val expected = true
        assertThat(expected).isTrue()
    }
}
